main.tex

```diff
@@ -777,13 +818,9 @@
 
 \end{enumerate}
 
-\item  An Owner or Occupier of a Lot without the consent of the Owners Corporation must not move any bulky goods or heavy goods, into or out of the Development except between the hours of 9.00AM and 5.00PM on Business Days and then only when the lift protection covers have been installed.
-
-\item  An Owner or Occupier must not hold the lift doors open and/or prevent the lift doors from closing for any lengthy periods of time which interferes with the normal operation of the lifts.
```

Comment:
Calling the lift at all is something that interferes with the operation of the lifts.